dist/Data-Dumper/Dumper.pm

```diff
@@ -1416,8 +1416,9 @@ distribution for more examples.)
 =head1 BUGS
 
 Due to limitations of Perl subroutine call semantics, you cannot pass an
-array or hash.  Prepend it with a C<\> to pass its reference instead.  This
-will be remedied in time, now that Perl has subroutine prototypes.
+array or hash.  Prepend it with a C<\> to pass its reference instead:
+Dumper C<\>(%a, @b, $c), C<\>%e, C<\>@f, C<\>$g.
+This will be remedied in time, now that Perl has subroutine prototypes.
```

Comment:
Who is going to remedy this? 
At what time? 
How do prototypes remedy with this?
Are you thinking of signatures?

Reply:
Perhaps that's a question for the original commit including that statement? Althought hisotry seems to stop here: https://github.com/Perl/perl5/commit/823edd999e1a252d36bcee7f4b8cc4bb197530e3

It seems however to refer to standard prototypes and not the new signatures.

Reply:
My point was more that the vibes are wrong. Optimistic use of future tense and prototypes really don't mix in `$current_year`

Reply:
It's definitely not a change we will be making. That sentence should be removed.

Reply:
Oh yes please remove it or whatever and also tweak my version if you want I'm very welcome.

I guess I'll learn what a version bump is a little later. But just go right ahead.